README.md

```diff
@@ -1 +1,24 @@
-# hello-world-react-rails
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
```

Comment:
- [ ] I would recommend making the README professional with all the necessary details like title, description, steps to clone and run the project locally with proper step by step command, author details, etc.. You can follow the [template](https://github.com/microverseinc/readme-template) given by the microverse as well.